README.md

```diff
@@ -37,40 +37,33 @@ The OpenSSH option `AuthorizedKeysCommand` cannot have any arguments
 specified following the command, and we need to pass options to
 specify which organization and team is used, as well as which user
 should be authenticated using keys in GitHub, so we will create a
-wrapper script for openssh-github-keys. Additionally
-`openssh-github-keys` will need your `GITHUB_TOKEN` which may be
-specified as a variable in the wrapper script, or in a
-[dotenv file](https://github.com/stackbuilders/dotenv-hs). Your
-wrapper script may look like the one below, which you could create as
-`/usr/local/bin/openssh-github-keys-wrapper`:
+configuration file for openssh-github-keys. Additionally
+`openssh-github-keys` will need your `GITHUB_TOKEN` which must be
```

Comment:
Actually it must be specified *either* in that file, or on the command line, right?